src/Main.elm

```diff
@@ -61,13 +61,75 @@ executeAdd model =
     }
 
 
+executeSubtract : Model -> Model
+executeSubtract model =
+    { model
+        | display = String.fromFloat <| model.valueA - model.valueB
+        , valueA = 0
```

Comment:
I would rename `valueA`, `actionA`, `valueB`, and `actionB`. If these are previous values/actions and current values/actions, or old and new, or somesuch, then name them that.